### DISCUSSION TITLE
插件配置中input是否支持在一个配置中可配置多个吗？

### DISCUSSION THREAD

**Discussion by ZZZZZZTong** [alibaba/ilogtail#1207] [Help] [answered]
我尝试在用户配置的一个配置文件里配置多个input插件导致失败
我查看了用户手册，也查看了代码文件里的example_config。均没有在一个inputs:下配置多个Type的例子
不过有配置多个processors,flushers 的例子
我参考了这两个的例子，写出inputs的配置如下：
enable: true
inputs:
 - Type: service_input_example
 - Type: file_log         # 文件输入类型
   LogPath: .              # 文件路径Glob匹配规则
   FilePattern: simple.log # 文件名Glob匹配规则

flushers:
  - Type: flusher_stdout    # 标准输出流输出类型
    OnlyStdout: true
其中service_inout_example 和file_log 插件在一个配置文件中进行单个配置都可正常运行。但合在一起就都不能运行了。
麻烦了解的大佬告知一下。。

**Comment by 7y-9** [comment]
使用多个input插件可以配置多个配置文件的，可以描述一下使用场景吗？

**Reply by ZZZZZZTong** [reply]
目前没有使用场景，就是自己在做测试。使用多个的插件时时在默认的情况下只有input里属性不同，flushers都是一样的，
在用户手册的“插件系统”这一节 看到介绍说 “Input: 作为输入层，一个配置可以同时配置多个 input，它们可以同时工作，采集所得到的数据将递交给 processor。”
我就尝试在一个配置文件里写多个input看看能否成功。

**Comment by TaylorJJL** [comment]
目前我也有遇到相同情况，请问您有遇到解决方式吗？如有麻烦能否告知，十分感谢。

**Comment by Takuka0311** [comment, marked as answer]
目前Input还不支持配置多个，文档会在近期修正，造成误导了很抱歉。
多Input存在开发计划（stage 2）：https://github.com/alibaba/ilogtail/discussions/855，目前进展是stage 1接近完成，但距离stage 2完成还有一段时间
